### DISCUSSION TITLE
Teams citations are truncated

### DISCUSSION THREAD

**Discussion by Maki-DS** [microsoft/teams-sdk#2191] [Q&A]
I used the deploy to teams app option in Azure AI Studio and the citations in the teams UI differ from those in AI Studio.

In AI Studio, the citation is clickable allowing you to read the whole chunk. In Teams, this is limited to a ~30-word hover window that cannot be expanded. I have investigated the POWERED-BY-AI docs and although I cannot see the SayCommand.ts in my project. When looking at `teams-ai/src/types
/ClientCitation.ts` in the repo I know that type DigitalDocument is required and the abstract cannot exceed 480 characters.

Can I circumvent this restriction, have it be expandable, or have it as a separate collapsable message?

**Comment by corinagum** [comment]
the 480 character limit is enforced by the Teams client, not our SDK. Unfortunately, there is no workaround.
